package.json

```diff
@@ -68,13 +72,13 @@
   },
   "scripts": {
     "build:evergreen": "node .evergreen/generate_evergreen_tasks.js",
-    "build:types": "tsc",
+    "build:types": "gulp -f test/tools/ts-pipeline.js",
     "check:atlas": "node test/tools/atlas_connectivity_tests.js",
     "check:bench": "node test/benchmarks/driverBench",
     "check:coverage": "nyc mocha --timeout 60000 --recursive test/functional test/unit",
     "check:lint": "eslint -v && eslint index.js lib test",
     "check:test": "mocha --recursive test/functional test/unit",
-    "check:types": "tsc -p tsconfig.check.json",
+    "check:types": "tsc -p src",
```

Comment:
if you go with my other comment on the scripts lets change to `check:ts` and regardless the command can be `tsc --noEmit`, right now this just crashes.

Reply:
oops thanks for catching that, left it in when I was playing around